lineage/figures/figure12.py

```diff
@@ -0,0 +1,49 @@
+""" This file depicts the distribution of phase lengths versus the states. """
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import itertools
+
+from ..data.Lineage_collections import Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM, Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM
+from .figureCommon import getSetup, subplotLabel
+from .figure11 import lpt12_g1, lpt12_g2, gmc12_g1, gmc12_g2
+
+
+def makeFigure():
+    """
+    Makes figure 12.
+    """
+    ax, f = getSetup((13.2, 6.66), (2, 4))
+    subplotLabel(ax)
+
+    concs = ["cntrl", "Lapt 25uM", "Lapt 50uM", "Lapt 250uM", "cntrl", "Gem 5uM", "Gem 10uM", "Gem 30uM"]
+
+    LAP = pd.DataFrame(columns=["state", "phase"])
+    GEM = pd.DataFrame(columns=["state", "phase"])
+    for i in range(4):
+        # lapatinib
+        LAP["state "+str(concs[i])] = [a[0] for a in lpt12_g1[i]] + [a[0] for a in lpt12_g2[i]]
```

Comment:
Where does this get filtered for state? Indexing operates on the position in a list, not on the value.

Reply:
Pretty sure the problem is here.

Reply:
Wait... I'm not definitively sure this is wrong, but why don't you just make the Pandas table in the first place, rather than storing all these lists and then reformatting them?

Reply:
Hmm.. yes I didn't realize I am redoing it. I imported the values from the `fitting` in figure11.py so figure 11 and 12 to be from the same fit (not completely sure it is actually happening, though). I will create the dataframe in figure11 and import that.